lib/Mojo/Util.pm

```diff
@@ -790,9 +790,9 @@ Create anonymous scope guard object that will execute the passed callback when t
 
 =head2 secure_compare
 
-  my $bool = secure_compare $str1, $str2;
+  my $bool = secure_compare $str, $secret;
```

Comment:
The choice of variable names tells me nothing.

Reply:
I've updated the POD accordingly.